packages/sdk/src/mint/mint.ts

```diff
@@ -61,6 +62,7 @@ export const mint = (
       owner_id: ownerId,
       metadata: {
         reference: reference,
+        media: media,
```

Comment:
Thoughts on extending this capability to other/all on chain standardized NEP 171 options?